app/ts-store/run/server_test.go

```diff
@@ -385,7 +385,7 @@ func (client *MockMetaClient) MarkDatabaseDelete(name string) error {
 func (client *MockMetaClient) MarkRetentionPolicyDelete(database, name string) error {
 	return nil
 }
-func (client *MockMetaClient) MarkMeasurementDelete(database, mst string) error {
+func (client *MockMetaClient) MarkMeasurementDelete(database, mst, rpName string) error {
```

Comment:
instead 'rpName' with rpname may be better, or 'rp_name'

Reply:
`rp_name` should not be used, refer https://google.github.io/styleguide/go/decisions#underscores.
`rpName` and `rpname` is acceptable, but for consistency with `database` and `mst`, `rp` or `retentionPolicy` is better, and `policy` is already used in the code, it is also an option.